Pipfile

```diff
@@ -31,6 +31,7 @@ pytest-flask = "~=0.10"
 pytest = "~=3.0"
 pytest-cov = "~=2.7.1"
 coveralls = "~=2.0"
+lovely-pytest-docker = "~=0.2.0"
```

Comment:
this change is already in #449, so perhaps we should merge #449 first and then double check what happens when rebasing, i.e. make sure nothing gets undone! Also we should get rid of the `Pipfile.lock` in this PR and replace it with the one coming from #449 after rebasing, see my comments there.